sample_config/config_korean.yaml

```diff
@@ -0,0 +1,50 @@
+node_lines:
+- node_line_name: retrieve_node_line
+  nodes:
+  - node_type: retrieval
+    modules:
+    - module_type: vectordb
+      embedding_model: openai
+      top_k: 3
+    strategy:
+      metrics:
+      - retrieval_f1
+      - retrieval_recall
+      - retrieval_precision
```

Comment:
How about just use `metrics: [ retrieval_f1, retrieval_recall, retrieval_precision ]`
Is it too long?